rke/structure_rke_cluster.go

```diff
@@ -88,6 +88,10 @@ func flattenRKECluster(d *schema.ResourceData, in *cluster.Cluster) error {
 
 	d.Set("dind", in.DinD)
 
+	if _, ok := d.Get("enable_cri_dockerd").(bool); ok && in.EnableCRIDockerd != nil {
+		d.Set("enable_cri_dockerd", in.EnableCRIDockerd)
```

Comment:
Should this be `*in.EnableCRIDockerd`? Asking because it looks like that's what `ignore_docker_version` is doing below.

Reply:
Yes this needs to be `*in.EnableCRIDockerd.` They are both *bool types. Docs https://pkg.go.dev/github.com/rancher/rke/types@v1.3.3#RancherKubernetesEngineConfig.EnableCRIDockerd